app/resources/api/v2/request_resource.rb

```diff
@@ -24,6 +24,7 @@ class RequestResource < BaseResource
       has_one :primer_panel
       has_one :pre_capture_pool
       has_many :poly_metadata, as: :metadatable, class_name: 'PolyMetadatum'
+      has_one :asset
```

Comment:
what is asset here? requests are referenced from multiple labware receptacles as the request is passed down from parent to child during transfers, it's not just linked to one receptacle

Reply:
You are right, this can be removed I think I was using it when experimenting with ways of retrieving the data but its no longer needed.